content/vendor/cli/channel delete.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2021-09-10
+linktitle: "channel rm"
+title: "channel rm"
+weight: 90110
+draft: false
+---
+
+Delete (archive) a channel.
+
+### Usage
+```bash
+replicated channel rm CHANNEL_ID
+```
+
+Only the channel ID will work, not the name.
```

Comment:
**Note:** You can only use the channel ID to delete the channel, not the channel name.